helmfile/charts/gatekeeper/templates/policies/tests/disallow-localhost-seccomp.rego

```diff
@@ -0,0 +1,571 @@
+package test.k8sdisallowlocalhostseccomp
+
+import data.k8sdisallowlocalhostseccomp
+
+#
+# Help functions
+#
+generate_pod_seccomp(type) = obj {
+    obj := {
+        "review": {
+            "object": {
+                "metadata": {
+                    "name": "test-pod"
+                },
+                "kind": "Pod",
+                "spec": {
+                    "securityContext": {
+                        "seccompProfile": {
+                            "type": type
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+generate_resource_controller_seccomp(kind, type) = obj {
+    obj := {
+        "review": {
+            "object": {
+                "metadata": {
+                    "name": "test-object"
+                },
+                "kind": kind,
+                "spec": {
+                    "template": {
+                        "spec": {
+                            "securityContext": {
+                                "seccompProfile": {
+                                    "type": type
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+generate_cronjob_seccomp(type) = obj {
+    obj := {
+        "review": {
+            "object": {
+                "metadata": {
+                    "name": "test-object"
+                },
+                "kind": "CronJob",
+                "spec": {
+                    "jobTemplate": {
+                        "spec": {
+                            "template": {
+                                "spec": {
+                                    "securityContext": {
+                                        "seccompProfile": {
+                                            "type": type
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+generate_pod_container_seccomp(containers) = obj {
+    obj := {
+        "review": {
+            "object": {
+                "metadata": {
+                    "name": "test-pod"
+                },
+                "kind": "Pod",
+                "spec": {
+                    "containers": containers
+                }
+            }
+        }
+    }
+}
+
+generate_rc_container_seccomp(kind, containers) = obj {
```

Comment:
Just to make it 100% maybe also add the daemon and deployment?

Reply:
Do you want a test with only a deployment and daemonset instead of using the [array of kinds](https://github.com/elastisys/compliantkubernetes-apps/pull/1865/files#diff-5f51324d7de735e2255dee045eedb3166b7df7830abf3e9d8c6c89748191cd8eR564-R571)?

Reply:
I missed the kind list. So ignore this comment :slightly_smiling_face: